mobile/lib/blocs/nearby_location/nearby_location_bloc.dart

```diff
@@ -57,21 +57,19 @@ class NearbyLocationBloc
     List<AirQualityReading> airQualityReadings =
         HiveService().getAirQualityReadings();
 
-    nearByAirQualityReadings = nearByAirQualityReadings
-        .map((element) {
-          AirQualityReading referenceReading = airQualityReadings.firstWhere(
-            (reading) => reading.referenceSite == element.referenceSite,
-            orElse: () => element,
-          );
-
-          return element.copyWith(
-            pm10: referenceReading.pm10,
-            pm2_5: referenceReading.pm2_5,
-            dateTime: referenceReading.dateTime,
-          );
-        })
-        .toList()
-        .sortByDistanceToReferenceSite();
+    nearByAirQualityReadings = nearByAirQualityReadings.map((element) {
+      AirQualityReading referenceReading = airQualityReadings.firstWhere(
+        (reading) => reading.referenceSite == element.referenceSite,
+        orElse: () => element,
+      );
+
+      return element.copyWith(
+        pm10: referenceReading.pm10,
+        pm2_5: referenceReading.pm2_5,
```

Comment:
I am wondering, since PM2_5 is our primary and perhaps only concentration we consider in all our public facing apps, could we just focus on it and do away with other fields/concentrations? Even at API level? @NoahNsimbe

Reply:
We can focus on PM2.5 in the app but I think we should not remove PM10 at the API level.  @Baalmart

Reply:
Alright, thanks @NoahNsimbe